src/analyzer/analyzerrhythm.cpp

```diff
@@ -22,7 +22,7 @@ namespace {
 // TODO: kStepSecs and the waveform sample rate of 441
 // (defined in AnalyzerWaveform::initialize) do not align well and thus
 // generate interference. Currently we are at this odd factor: 441 * 0.01161 = 5.12.
-// This should be adjusted to be an integer.
+// This should be adjusted to be an integer.0.0113378684807
```

Comment:
typo?

Reply:
I was experimenting with some other values for kStepSecs, but 0.01161 seems like a magic number and works best that any other. Probably, because that's what was used to tune the algorithm in the first place.

A smaller number, around half of that, like around 0.006, can lead to a more tight grid but will double the tempo by detecting false beats in between them.

That one, 0.0113378684807, * 441 = 5.  Left it there for reference later.

Reply:
I am still confused. 0.0113378684807 is not 0.01161.

Reply:
Exactly, I was experimenting with other values, different than 0.01161. 
Since @daschuer path to compute the window size based on a max value, we don't need this to lead to a power of 2 anymore. Still, 0.01161 seems to works best, but we might be able to change that by digging into tempotrackv2 code.
That value I left there is a candidate that we can use that fulfills the comments Daniel left to align it with the waveform.

Reply:
Either remove the magic number 0.0113378684807 from the comment or explain why it is there.